clippy_utils/src/msrvs.rs

```diff
@@ -12,6 +12,7 @@ macro_rules! msrv_aliases {
 
 // names may refer to stabilized feature flags or library items
 msrv_aliases! {
+    1,99,0 { LET_ELSE }
```

Comment:
Isn't let_else already stable in `1.59`?

Reply:
No: https://github.com/rust-lang/rust/pull/93628